back/app/jobs/log_activity_job.rb

```diff
@@ -88,4 +89,10 @@ def trigger_track_activity_job(activity, original_item)
     # expensive
     TrackEventJob.perform_later(activity) unless original_item.is_a?(Notification)
   end
+
+  def update_user_last_acted_at(user, acted_at)
+    return unless acted_at
+
+    user&.update!(last_acted_at: acted_at)
```

Comment:
I'm wondering if this can happen
```
LogActivityJob A is scheduled
# a day passes
LogActivityJob B is scheduled
LogActivityJob B is performed
LogActivityJob A is performed
```
In this case, `last_acted_at` will not be the latest. Maybe adding a simple check like `if acted_at > last_acted_at` could be useful.

Reply:
Yes, and I had thought of this, then totally forgotten. Thanks!